ui/ui-components/components/source/FormMappingSelector.tsx

```diff
@@ -113,7 +113,7 @@ const FormMappingSelector: React.FC<Props> = ({
             as="select"
             required
             disabled={disabled || !previewColumns.length}
-            defaultValue={columnName || previewColumns?.[0] || ""}
+            defaultValue={""}
```

Comment:
I think you can remove this entirely?

Reply:
Hmm.  Removing this default makes the form (a) not show the intended default (it shows blank instead of "Select an Option") and (b) results in the form defaulting to the first option rather than "Select an Option" when the table is selected.

Reply:
🤷 Works for me then